ocs_ci/utility/lvmo_utils.py

```diff
@@ -56,3 +63,66 @@ def lvmo_health_check(tries=20, delay=30):
         delay=delay,
         backoff=1,
     )(lvmo_health_check_base)()
+
+
+def get_disks_by_path():
```

Comment:
From what I understand from what the function does, it makes sense to me to rename it to something like:
`get_sno_node_disk_paths`